library/core/src/slice/mod.rs

```diff
@@ -3994,6 +3994,34 @@ impl<T> [T] {
     }
 }
 
+#[cfg(not(bootstrap))]
+#[lang = "array_slice"]
+impl<T, const N: usize> [[T; N]] {
+    pub fn flatten(&self) -> &[T] {
```

Comment:
Note that you're going to need `#[unstable]` attributes on these https://std-dev-guide.rust-lang.org/feature-lifecycle/new-unstable-features.html

Reply:
Was aware of that, just wasn't sure what to put in them. Was going to ask on Zulip